tests/builders/tests/test-helper.js

```diff
@@ -6,13 +8,18 @@ import assertAllDeprecations from '@ember-data/unpublished-test-infra/test-suppo
 import configureAsserts from '@ember-data/unpublished-test-infra/test-support/qunit-asserts';
 import customQUnitAdapter from '@ember-data/unpublished-test-infra/test-support/testem/custom-qunit-adapter';
 
+import Application from '../app';
+import config from '../config/environment';
+
 // Handle testing feature flags
 if (QUnit.urlParams.enableoptionalfeatures) {
   window.EmberDataENV = { ENABLE_OPTIONAL_FEATURES: true };
 }
 
 configureAsserts();
 
+setApplication(Application.create(config.APP));
```

Comment:
in follow up we should eliminate this and the new app files in favor of in-test-file registration